paddle/fluid/framework/ir/fc_gru_fuse_pass.cc

```diff
@@ -350,7 +350,7 @@ void FCGRUFusePass::ApplyImpl(ir::Graph* graph) const {
       graph, name_scope_, param_scope(), true /*with_fc_bias*/);
 
   AddStatis(fusion_count);
-  if (!Has("disable_logs") || !Get<bool>("disable_logs"))
+  if ((!Has("disable_logs") || !Get<bool>("disable_logs")) && fusion_count > 0)
```

Comment:
What do you think about it?
```suggestion
  const auto enabled_logs = !Has("disable_logs") || !Get<bool>("disable_logs");
  if (enabled_logs && fusion_count > 0)
```

Reply:
I think we might be extracting this to a separate function in future as @Silv3S proposed and there it could be good idea to do it as you suggest. For now, I think it's fine both ways. I just made this quickly while working on something else and I decided to share it. What do you think about leaving it as it is for now and rewriting this later?
Here is the discussion:
https://github.com/PaddlePaddle/Paddle/pull/46917#discussion_r993268723

Reply:
@Silv3S liked your suggestion so I'm fine with it as well.
We just need to remember about it.